channels/alpha

```diff
@@ -7,6 +7,8 @@ spec:
     - name: kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2017-01-09
       providerID: aws
       kubernetesVersion: ">=1.5.0"
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
```

Comment:
I don't like how we have to manage different projects in the GCE code paths 🤔 I don't know if there's any way around this though.